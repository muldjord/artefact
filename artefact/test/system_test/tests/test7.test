INPUT="insertlinses.xml"
EXPECT="test7_expected.xml"
FILTER="cat"
TITLE="Checking that different data entries do not get mixed when querying with filter latest"
