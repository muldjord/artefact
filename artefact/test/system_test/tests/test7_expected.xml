<?xml version='1.0' encoding='utf-8'?>
<results>
  <result id="0" errcode="0" errstring="Data Success" class="status"
          deviceid="" devicetype="" timestamp="">
  </result>
  <result id="1" errcode="0" errstring="Data Success" class="status"
          deviceid="" devicetype="" timestamp="">
  </result>
  <result id="2" errcode="0" errstring="" class="alcon"
          deviceid="CHANGE_ME" devicetype="barcode" timestamp="">
    <group name="barcode">
      <value name="vendor">Alcon</value>
      <value name="type">alcon</value>
      <value name="rawdata">+H530SA60AT120P16
+1124311495700665</value>
      <value name="serial">+1124311495700665</value>
      <value name="power">12.0</value>
      <value name="model">SA60AT</value>
    </group>
  </result>
</results>
